usaspending_api/accounts/tests/test_tas_autocomplete.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+import pytest
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.accounts.models import TreasuryAppropriationAccount
+from usaspending_api.common.tests.autocomplete import check_autocomplete
+
+
+@pytest.fixture
+def tas_data(db):
+    mommy.make(
+        TreasuryAppropriationAccount,
+        tas_rendering_label='zzz',
```

Comment:
Thanks for adding tests, but I would like to point out that your test `tas_rendering_labels` are not nearly as amusing as your fake agency names, which is a little disappointing tbh